release-notes/v1_62.md

```diff
@@ -81,6 +81,24 @@ There is a new setting `html.completion.attributeDefaultValue` that lets you cho
 
 Work continues on the [GitHub Pull Requests and Issues](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-pull-request-github) extension, which allows you to work on, create, and manage pull requests and issues. Check out the [changelog for the 0.32.0](https://github.com/microsoft/vscode-pull-request-github/blob/main/CHANGELOG.md#0320) release of the extension to see the highlights.
 
+### Jupyter
+
+**Kernels**
+
+The list of kernels displayed in the kernel picker is now grouped to make it easier for one to identify and find kernels belonging to a specific Python envoironment.
```

Comment:
This sentence reads kind of funny to me.. I would recommend changing the order. 

"In order to make it easier to identify kernels, they are now grouped in the kernel picker" 

Also, environment typo